### PR TITLE
feat: add mdx support

### PR DESCRIPTION
**Pull request by oscar-cyou**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🚀 Application Performance Improvements
- [ ] 🔧 Code Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert
- [ ] ♿ Accessibility Improvements
- [ ] 📦 Dependencies

## Description

mdx support added

## Screenshots/Recordings

Screenshot 2024-07-28 at 9 59 57 p m

## Related Tickets

https://github.com/oscar-cyou/oscar-cyou-website/issues/3

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed

## Added to documentation?

- [ ] 📜 The README.md has been updated
- [ ] 📓 Internal Documentation
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by bytebebop** [comment]
no tests added?

File: src/app/blog/my-second-post/page.mdx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+export const metadata = {
+  title: post.title,
+  description: post.description,
+};
+
+export default (props) => <PostLayout post={post} {...props} />;
+
+## This is and h2
+
+Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus.
```
Comment (bytebebop) [inline src/app/blog/my-second-post/page.mdx:19]: will this text stay?

**Review by bytebebop — COMMENTED** [review]
will this text stay?

File: src/app/blog/my-second-post/page.mdx (hunk 1 above)
Comment (oscar-cyou) [reply src/app/blog/my-second-post/page.mdx:19]: For the moment yes, it was only demonstrative but it will be removed in the next pr

**Pull request merged**